### PR TITLE
Fix wells, tidy API examples

### PR DESCRIPTION
**Pull request by BenJam**
Fix up three different well sizes and uses them in project#show,
api#index and pages#data.

### CODE REVIEW

**Comment by andrew** [comment]
@BenJam can you rename the `small` and `large` classes for wells to be called `well-small` and `well-large`, similar naming to bootstrap, should help avoid clashes in the future if someone else makes a `small` class for something else that affects the wells by accident

**Comment by andrew** [comment]
@BenJam we'll need both `well` and `well-small` in the HTML for each `<pre>`

File: app/views/projects/index.html.erb [hunk 1]
```diff
@@ -16,18 +16,18 @@
...
+<div class='row cta'>
   <div class='col-md-3'>
     <div class="panel panel-default">
       <div class="panel-heading">
         <h2 class="panel-title"><%= fa_icon 'binoculars' %>&nbsp;Discover new software</h2>
       </div>
       <div class="panel-body">
         <p>
-          Search <%= number_with_delimiter(Project.total) %> projects by <%= link_to 'licence'.html_safe, licenses_path %>, <%= link_to 'language'.html_safe, licenses_path %> or <%= link_to 'keyword'.html_safe, keywords_path %>, or explore new, trending or popular projects.
+          Search <%= number_to_human(Project.total) %> projects by <%= link_to 'licence'.html_safe, licenses_path %>, <%= link_to 'language'.html_safe, licenses_path %> or <%= link_to 'keyword'.html_safe, keywords_path %>, or explore new, trending or popular projects. And some text to make this box longer
```
Comment (tombh) [inline app/views/projects/index.html.erb]: Long line, but at the end it says, 'And some text to make this box longer', hehe :D

Comment (tombh) [inline app/views/projects/index.html.erb]: PS, I forget, who gets notified of comments? Only the commiter? Or subsrcibers to the thread?

Comment (andrew) [inline app/views/projects/index.html.erb]: Thread subscribers I think, hey tom 👋

**Pull request merged**